src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -639,6 +639,10 @@ public enum Mixins {
             .addMixinClasses("extrautilities.MixinBlockDrum").setSide(Side.BOTH).setPhase(Phase.LATE)
             .setApplyIf(() -> FixesConfig.fixExtraUtilitiesDrumEatingCells)
             .addTargetedMod(TargetedMod.EXTRA_UTILITIES)),
+    FIX_GREENSCREEN_MICROBLOCKS(new Builder("Fix extra utilities Lapis Caelestis microblocks")
+            .addMixinClasses("extrautilities.MixinFullBrightMicroMaterial").setSide(Side.BOTH).setPhase(Phase.LATE)
```

Comment:
nit: Is this really BOTH? and not CLIENT?

Reply:
Hmm. You're probably right. They do make the blocks emit light, but I guess this is only how they are rendered. I'll change to CLIENT.